( 2021-01-06 04:33:44 (MSK) ) Сработала функция errorCatcher; Сбой в работе сайта. Код ошибки/Класс ошибки: 2; Информация об ошибке: Invalid callback CloudStore\App\Engine\ActiveRecord\Tables\Context::exceptionHandler, cannot access private method CloudStore\App\Engine\ActiveRecord\Tables\Context::exceptionHandler(); Файл: Unknown; Строка: 0
( 2021-01-06 04:33:44 (MSK) ) Сработала функция fatalErrorCatcher; Сбой в работе сайта. Код ошибки/Класс ошибки: 4; Информация об ошибке: syntax error, unexpected '}'; Файл: D:\Google Drive\Programming\Projects\Local\modware.ru\app\mvc\admin\models\ModelAdmin.php; Строка: 355
